### PR TITLE
docs: Update minimum Node.js requirements documented on our website to match current policy

### PR DESCRIPTION
**Pull request by Josmithr**
Updates a template file that gets embedded in relevant pages.

### CODE REVIEW

**Review by CraigMacomber — APPROVED** [review]

File: docs/_includes/node-versions.md [hunk 1]
```diff
@@ -1 +1 @@
-Fluid supports Node.js LTS versions 18 or above.
+Fluid supports Node.js LTS versions 20 or above.
```
Comment (CraigMacomber) [inline docs/_includes/node-versions.md:1]: thats typically used in fluid client specific contexts right? And is referring to testing and developeing  client code?

Comment (CraigMacomber) [reply docs/_includes/node-versions.md:1]: Production use of fluid client is typically in a browser and not node, and doesn't have any node dependencies, so its not too picky, but if we are talking about runing services, we might want to be more careful.

Comment (Josmithr) [reply docs/_includes/node-versions.md:1]: Yes. Good point. We may want to update this phrasing. Maybe:
```suggestion
The Fluid Framework supports building and testing under Node.js LTS versions 20 or above.
```

Comment (Josmithr) [reply docs/_includes/node-versions.md:1]: @kashms interested in your thoughts on wording, as always 🙂

Comment (Josmithr) [reply docs/_includes/node-versions.md]: I went with the above wording for now. We can always update this further in the future.

**Pull request merged**